requirements.txt

```diff
@@ -0,0 +1,5 @@
+construct
```

Comment:
It's better not to introduce files like this under root dir.

Reply:
As I cannot put comments on the binary data files, I'll let them here:

* following the convention of the GATK repo, files larger than 200k is better to be handled with git lfs
* I'd propose moving "test/test_data/tiny.bam" into "test/test_data/for_scripts/tiny.bam" (and the associated BAI); related, the test data for WDL/workflows might be moved to "test/test_data/workflows" (but can/should be done in a separate PR).
* I'd prefer a more specific name than "tiny.bam"

Reply:
Done - renamed to test/test_data/for_scripts/shard_bam_test_file.bam and is now LFS-tracked.

Reply:
I think most Python projects place the requirements.txt file at the root of the repo, optionally in addition to test-requirements.txt and/or dev-requirements.txt files.  I'm being a bit lazy here in that the requirements.txt file now lists requirements for scripts within Docker images and our CI/CD system.  I anticipate more scripts over time (e.g. scripts to simplify running the monitoring tools, or putting input file metadata into BigQuery for later analysis, etc.).

Reply:
I'm not quite understanding this.
Do you mean that later PRs should break down this single `requirements.txt` as 
> test-requirements.txt and/or dev-requirements.txt files

If that's the case, I agree, with one caveat: I view this repo as pipeline only, and try hard not to put any scripts other than easy, 1-time use utility scripts here. Tests of course are different.

Reply:
Yep, I agree with this.